connectors/sources/graphql.py

```
@@ -487,15 +496,37 @@ async def validate_config(self):
                 msg = f"Error while processing configured GraphQL variables. Exception: {exception}"
                 raise ConfigurableFieldValueError(msg) from exception
 
-        for graphql_object in self.graphql_client.graphql_object_list:
-            if not self.check_field_existence(ast=ast, field_path=graphql_object):
+        if graphql_object_list:
```

Else, should we throw an error since nothing will get indexed?